src/ExtractGQL.ts

```diff
@@ -61,7 +61,7 @@ export class ExtractGQL {
   public queryTransformers: QueryTransformer[] = [];
 
   // The file extension to load queries from
-  public extension: string;
+  public extension: Array<string> = ['graphql'];
```

Comment:
We should rename this to be a plural thing rather than leave it "extension" since it is now an array

Reply:
The only reason I didn't change this is the `CLI` arg is `extension` - and since renaming the arg would be a breaking change, elected to go with consistency.

Obv easy / happy to do this, but wanted to explain my thinking.